examples/phewas_scatter.html

```diff
@@ -194,6 +286,50 @@ <h6 style="float: right;"><a href="../index.html">&lt; return home</a></h6>
           plot.clearPanelData(null, "reset");
         }
 
+        function showInvalidVariant() {
+          document.querySelector("#error-variant").style.display = "";
+        }
+
+        function hideInvalidVariant() {
+          document.querySelector("#error-variant").style.display = "none";
+        }
+
+        function checkVariant(variant) {
+          var [chrom,pos,ref,alt] = variant.split("_");
+          var alleles = ["A","C","G","T"];
+          if (!Number.isInteger(parseInt(pos))) { showInvalidVariant(); return false} 
+          else if (!alleles.includes(ref)) { showInvalidVariant(); return false }
+          else if (!alleles.includes(alt)) { showInvalidVariant(); return false }
+          else { return true; }
+        }
+
+        function jumpToVariant() {
+          var input_variant = document.querySelector("#input-variant");
+          if (input_variant) {
+            var jmp_variant = input_variant.value
+            if (jmp_variant) {
+              jmp_variant = jmp_variant.trim().replace("/","_").replace(":","_");
+            } else {
+              jmp_variant = input_variant.placeholder.trim().replace("/","_").replace(":","_")
+            }
+
+            if (checkVariant(jmp_variant)) {
+              hideInvalidVariant();
+              loadPheWAS(jmp_variant)
+            }
+          }
+        }
+
+        // Control to jump to any variant
+        document.querySelector("#button-jump").addEventListener("click",event => {
```

Comment:
I love ES6 arrow functions! Unfortunately LZjs doesn't transpile, so we've continued using old-style `function() {}` syntax to make sure the demo works in older browsers as well. In theory our codebase supports at least IE11 and quite likely older.

(This note also applies to destructuring assignments and a few other items; I'll make these tweaks after the merge)